src/ODriveSTM32CAN.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "ODriveCAN.h"
+#include <STM32_CAN.h>
+
+using CanMsg = CAN_message_t;
+
+// Must be defined by the application
+void onCanMessage(const CanMsg& msg);
+
+static bool sendMsg(STM32_CAN& can_intf, uint32_t id, uint8_t length, const uint8_t* data) {
+  CanMsg msg;
+  msg.id = id;
```

Comment:
The top three bits of `id` are reserved, with the MSB meaning "extended". Additionally, if data is null, it should send a remote request frame if the CAN lib supports it.

```suggestion
  msg.id = id & 0x1ffffff;
  msg.flags.extended = id & 0x80000000;
  msg.flags.remote = data;
```
(not tested)